Library/Formula/packer.rb

```diff
@@ -199,6 +199,7 @@ def install
     packerpath = buildpath/"src/github.com/mitchellh/packer"
     packerpath.install Dir["{*,.git}"]
     Language::Go.stage_deps resources, buildpath/"src"
+    mkdir_p buildpath/"bin"
```

Comment:
Didn't spend too much time trying to track it down, but the upstream build script was trying to copy to GOPATH/bin, which didn't exist in the buildpath... not sure when that change came in.